### ISSUE TITLE
Maven Repository

### ISSUE THREAD

**Issue by tolulopep** [svn2github/jrecord#1]
Can this library be put in maven respository? It is easier to pull like that and it increases awareness too.

**Comment by bmTas** [comment]
Yes it's a great Idea, I started to do it but kept getting interrupted. I really do not have the time to research what is involved + I do not use Maven or understand it.

Why not have a go your self and then tell me what is involved
